lab_5_scraper/scraper.py

```diff
@@ -206,12 +393,26 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    if base_path.exists():
+        if any(base_path.iterdir()):
+            shutil.rmtree(base_path)
+    base_path.mkdir(parents=True, exist_ok=True)
 
 
 def main() -> None:
     """
     Entrypoint for scrapper module.
     """
+    configuration = Config(path_to_config=CRAWLER_CONFIG_PATH)
+    prepare_environment(ASSETS_PATH)
+    crawler_recursive = CrawlerRecursive(config=configuration)
```

Comment:
for crawler recursive create a separate main - main2 and show all demonstration for recursive crawler there. do not call main2 in the if __name__ == "__main__", you will need it only for oral handover